cmd/exporter.go

```diff
@@ -51,6 +55,10 @@ func healthProbe(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func finalizer() {
+	fmt.Println(kFlag)
```

Comment:
We should not use `fmt` package to print anymore....
The PR #238 moved all prints and logs to `klog`, so for consistency we should use klog

Reply:
This also should be a more meaningful word, for example:
```golang
klog.InfoS("exiting...")
exitCode := 10
klog.FlushAndExit(klog.ExitFlushTimeout, exitCode)
```
As in the example of the library [here](https://pkg.go.dev/k8s.io/klog/v2#FlushAndExit).